src/plugins/intel_cpu/tests/functional/single_layer_tests/detection_output.cpp

```diff
@@ -111,6 +138,18 @@ class DetectionOutputLayerCPUTest : public testing::WithParamInterface<Detection
         return result.str();
     }
 
+    static void scale_input_shapes(std::vector<ov::test::InputShape> &inShapes) {
```

Comment:
Could you please add the comment why we have this function?

Reply:
added, thanks.